.pear/pear.sh

```diff
@@ -16,13 +16,7 @@ if [ "$SHA1" ]; then
     exit 0;
 fi
 
-coauthors="Co-authors:"
-n=1
-while read line; do
-    coauthors="${coauthors} $line,"
-    n=$((n+1))
-done < $filename
-coauthors="$(echo $coauthors | sed 's/,$//g')" # Remove trailing comma
+coauthors="Co-authors: $(paste -sd "," $filename | sed -e 's/,$//')"
```

Comment:
Nice! This looks like it works to me.

One thing I noticed is the previous output was: 

```
Co-authors: Ryan Wessel, Master Chief, Gandalf the Grey, Tony Stark
```

And with this change, it looks like this:

```
Co-authors: Ryan Wessel,Master Chief,Gandalf the Grey,Tony Stark
```

[In the matrix command](https://github.com/ryanjwessel/pear-cli/blob/main/src/commands/matrix.ts#L76), we split these strings by `, ` (note the space after the comma), so this would prevent the pairs from being split correctly.

Reply:
I'm not sure if there is a straightforward way to include the trailing space in this one-liner, or we could update the matrix command to no longer split by `, `.  It seems like updating the matrix command would be preferable?

It could be something like:

```js
body.split(CO_AUTHORS)[1].split(',').map(pair => pair.trim())
```

I think replacing new-lines in that command was a leftover from a previous attempt at doing this.

Reply:
Also, I realize it is a bit unclear, but this is the generated shell script file. The file that is used for publishing is found in `src/pear.sh`.